src/components/user-form/AddUserForm.js

```diff
@@ -33,14 +36,23 @@ class AddUserForm extends Component {
     }
 
     console.log(firstName, lastName, email);
-    // todo 1: использовать функцию onUserAdd для создания нового пользователя здесь
+    // donetodo 1: использовать функцию onUserAdd для создания нового пользователя здесь
     //  у этого объекта должны быть проперти first_name, last_name, address, email
-
-    // todo 3: зачистить форму, вызвав функцию onReset
+    onUserAdd && onUserAdd({
+      firstName, lastName, address, email
```

Comment:
I guess there are incorrect keys in user
for example `firstName` supposed to be `first_name `

Reply:
Does user appear in user's list?